frontend/src/pages/Admin/InventoryManagement.css

```diff
@@ -0,0 +1,45 @@
+.title{
+   text-align: center;
+    margin: 30px;
+}
+
+
+.container{
+    margin-top:30px;
+    margin-bottom: 50px;
+    display: flex;
+    justify-content:space-between;
+    align-items:center;
+    border-radius: 10px;
+    
+}
+
+.table-container{
+    
+    border-radius: 10px;
+    margin:20px;
+}
+
+.edit-button{
+    background-color: #000000;
+    border: none;
+    color: white;
+    padding: 10px 20px;
+    text-align: center;
+    display: inline-block;
+    font-size: 12px;
+    border-radius: 5px;
+    cursor: pointer;
+}
+
+.delete-button{
+    background-color: red;
+    border: none;
+    color: white;
+    padding: 10px 20px;
+    text-align: center;
+    display: inline-block;
+    font-size: 12px;
+    border-radius: 5px;
+    cursor: pointer;
+}
```

Comment:
We can remove these lines because `<Button>` component already defines styles for buttons